NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 ## Updates of Existing Functions
 
+* Updated arg_name parameter in `assert_filter_cond`, `assert_symbol`, and `assert_expr` with  `gsub("^enexpr\\((\\w+)\\)$", "\\1", rlang::caller_arg(arg))` argument so that  error messages return a quoted argument without `enexp()`, e.g. `filter_add` instead of `enexpr(filter_add)`.
```

Comment:
```suggestion
* Updated `arg_name` parameter in `assert_filter_cond`, `assert_symbol`, and `assert_expr` with  `gsub("^enexpr\\((\\w+)\\)$", "\\1", rlang::caller_arg(arg))` argument so that  error messages return a quoted argument without `enexpr()`, e.g. `filter_add` instead of `enexpr(filter_add)`.
```